src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,63 @@
+import cn from 'classnames';
+import { useParams } from 'react-router-dom';
+import { FC } from 'react';
+import { PersonLink } from './PersonLink';
+import { Person } from '../types';
+
+interface Props {
+  people: Person[]
+}
+
+export const PeopleTable:FC<Props> = ({ people }) => {
+  const { userSlug } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
```

Comment:
can we create an enum or variable here and map over it?

Reply:
Yes we can :)